docs/en/guides/action-recognition.md

```diff
@@ -0,0 +1,92 @@
+---
+comments: true
+description: Learn how to recognize actions in real-time using Ultralytics YOLOv8 for applications like surveillance, sports analysis, and more.
+keywords: action recognition, YOLOv8, Ultralytics, real-time action detection, AI, deep learning, video classification, surveillance, sports analysis
+---
+
+# Action Recognition using Ultralytics YOLOv8
+
+## What is Action Recognition?
+
+Action recognition involves identifying and classifying actions performed by objects (typically humans) in video streams. Using [Ultralytics YOLOv8](https://github.com/ultralytics/ultralytics/), you can achieve real-time action recognition for various applications such as surveillance, sports analysis, and more.
+
+## Advantages of Action Recognition
+
+- **Enhanced Surveillance:** Detect and classify suspicious activities in real-time, improving security measures.
+- **Sports Analysis:** Analyze player movements and actions to provide insights and improve performance.
+- **Behavior Monitoring:** Monitor and analyze behaviors in various settings, such as retail or healthcare.
+
+## Real World Applications
+
+|                                          Surveillance                                          |                                           Sports Analysis                                            |
+| :--------------------------------------------------------------------------------------------: | :--------------------------------------------------------------------------------------------------: |
+| ![Surveillance](https://github.com/RizwanMunawar/ultralytics/assets/62513924/surveillance.jpg) | ![Sports Analysis](https://github.com/RizwanMunawar/ultralytics/assets/62513924/sports_analysis.jpg) |
```

Comment:
these images does not exist. where should I upload them and what should be their size?

Reply:
@fcakyon These images not required for action recognition, I think so?

Reply:
Makes sense, removed that section.